services/movie.py

```diff
@@ -1,14 +1,21 @@
+from typing import Optional
+
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: Optional = None,
```

Comment:
It is also a good practice to specify what is expected in optional: `Optional[list] = None` for example. 
Also these lines are overindented

Reply:
![NVIDIA_Share_zSop26mCtA](https://github.com/mate-academy/py-tickets-and-orders/assets/13889031/9e1709d0-e8b3-4821-97b5-3c8ce3e9f6e4)
I don't understand where are they overintended, they are not repeatable in my code nomore